mod/groups/views/default/groups/invitationrequests.php

```diff
@@ -46,6 +63,14 @@
 		}
 	}
 	echo '</ul>';
+
+	if (!empty($count)) {
+		echo elgg_view('navigation/pagination', array(
```

Comment:
why not switch to `elgg_view('page/components/list')` here? Then there is no need to draw pagination yourself